tracking/models.py

```diff
@@ -8,6 +8,16 @@
 
 from django.conf import settings
 
+class MentorStudentCourseObserver(models.Model):
```

Comment:
I feel like this should be in seumich.models rather than in tracking. Tracking is more about event tracking, not about Mentor/Student relationships.

Reply:
The main reason I did not put this into seumich.models is because according to routers.py, seumich models are related to the seumich database(Oracle database), and I want this model to be inside the Student Explorer MySQL database. 

And this table serves also for the tracking purpose: we track whether we have added this mentor to what course and observer which student...

Reply:
You're completely right, it should probably be in management not in seumich. The tracking folder is really just for a very simple event log module and I'd like to replace it with the same logger that MyLA uses. This is just tracking module is just custom code for doing logging that's seemed very easily replaced with that package.

It could be in a new top level folder with the cron job called cron or jobs? I don't think there's any best practice for where to put that. I feel like management is where the local MySQL stuff has been going but can double check.

Reply:
The MySQL database contains:

auth_* (6 tables from django auth)
django_* (4 tables from various django packages)
feedback_feedback (From feedback)
management_* (4 tables from the management project)
registration_registrationprofile (From registration package)
tracking_event (From tracking)

So to me it maps closest to tables in management (since the UI may also want to present that the user was added to a site or not) or a new table that would be in a new package called jobs. That would prefix it to jobs_<something>

I guess it's fine to leave it and just move it it when we remove up tracking. . . My vote would be to move it all to management.